app/src/main/java/com/github/dedis/student20_pop/utility/ui/eventadapter/ExpandableListViewEventAdapter.java

```diff
@@ -174,7 +177,30 @@ public boolean hasStableIds() {
      * @return the View corresponding to the child at the specified position
      */
     @Override
-    public abstract View getChildView(int groupPosition, int childPosition, boolean isLastChild, View convertView, ViewGroup parent);
+    public View getChildView(int groupPosition, int childPosition, boolean isLastChild, View convertView, ViewGroup parent){
+        //TODO : For the moment, events are displayed the same if user is attendee or organizer,
+        // in the future it could be nice to have a pencil icon to allow organizer to modify an event
+        Event event = ((Event) getChild(groupPosition, childPosition));
+        String eventTitle = (event.getName() + " : " + event.getType());
+        String eventDescription = "Time : " + event.getTime() + "\nLocation : " + event.getLocation();
+
+        if (convertView == null) {
+            LayoutInflater inflater = (LayoutInflater) this.context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
+            convertView = inflater.inflate(R.layout.layout_event, null);
+        }
+        TextView eventTitleTextView = convertView.findViewById(R.id.event_title);
+        TextView descriptionTextView = convertView.findViewById(R.id.event_description);
+        eventTitleTextView.setText(eventTitle);
+        switch (event.getType()) {
+            case ROLL_CALL:
+                eventDescription += ("\nParticipants: " + event.getAttendees().size());
+                break;
+            default:
```

Comment:
Why are we not taking into account the other event types?

Reply:
Because they are not yet supported by the protocol, and the priority was the roll call event.

Reply:
Ok sounds good